engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/deployment/DeploymentPreparer.scala

```diff
@@ -12,7 +12,11 @@ import skuber.EnvVar.FieldRef
 import skuber.LabelSelector.IsEqualRequirement
 import skuber.apps.v1.Deployment
 import skuber.{Container, EnvVar, HTTPGetAction, LabelSelector, Pod, Probe, Volume}
+import DeploymentPreparer._
 
+object DeploymentPreparer {
+  val SharedConfigmapName = "shared"
```

Comment:
shared-logback ?

Reply:
I think it should be configurable... E.g. in helm we may want to prefix it with release name etc.